<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="VlcVideoWidget" name="video" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>3</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="time_bar">
      <item>
       <widget class="SeekSlider" name="position">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TimeLabel" name="timeLabel"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="control_bar">
      <item>
       <widget class="PlayButton" name="btn_play">
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_stop">
        <property name="icon">
         <iconset resource="../QtVlc/vlc/modules/gui/qt4/vlc.qrc">
          <normaloff>:/toolbar/stop_b</normaloff>:/toolbar/stop_b</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="quality"/>
      </item>
      <item>
       <widget class="SoundWidget" name="volume" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionBrowser"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionBrowser">
   <property name="text">
    <string>Browser</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VlcVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtVlcWidgets/VlcVideoWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SeekSlider</class>
   <extends>QSlider</extends>
   <header location="global">QtVlcWidgets/SeekSlider.h</header>
  </customwidget>
  <customwidget>
   <class>PlayButton</class>
   <extends>QToolButton</extends>
   <header location="global">QtVlcWidgets/SpecialButtons.h</header>
  </customwidget>
  <customwidget>
   <class>SoundWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtVlcWidgets/SoundWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TimeLabel</class>
   <extends>QLabel</extends>
   <header location="global">QtVlcWidgets/TimeLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../QtVlc/vlc/modules/gui/qt4/vlc.qrc"/>
 </resources>
 <connections/>
</ui>
